packages/controller/main.js

```diff
@@ -5679,6 +5669,36 @@ function init(compactGroupId) {
     process.on('unhandledRejection', exceptionHandler);
 }
 
+/**
+ * Parses out the rebuild path from an error log
+ *
+ * @param {string} text - log text
+ * @return {string | undefined}
+ * @private
+ */
+function _determineRebuildPathFromLog(text) {
+    // extract rebuild path - it is always between the only two single quotes
+    const matches = text.match(/'.+'/g);
+
+    if (matches && matches.length === 1) {
+        // remove the quotes
+        let rebuildPath = matches[0].replace(/'/g, '');
+        if (path.isAbsolute(rebuildPath)) {
+            // we have found a module which needs rebuild - we need to find deepest pack.json
+            rebuildPath = path.dirname(rebuildPath);
+            const rootDir = path.parse(process.cwd()).root;
+
+            while (rebuildPath !== rootDir) {
+                if (fs.pathExistsSync(path.join(rebuildPath, 'package.json'))) {
+                    return rebuildPath;
```

Comment:
Maybe add a path.normalize? Then you do not have a /bla/blubb/falsel/../../.. construct but something "real" :-)

Reply:
Or mnormalize directly on path.join

Reply:
The return of path join .. does not contain ..

Reply:
Cool ... learned something :-)